### PR TITLE
fix: setup.py requirements

### PR DESCRIPTION
**Pull request by Alchez**
```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/rohan/jhb/apps/shipment_management/setup.py", line 27, in <module>
        install_requires=[str(ir.req) for ir in requirements],
  File "/home/rohan/jhb/apps/shipment_management/setup.py", line 27, in <listcomp>
        install_requires=[str(ir.req) for ir in requirements],
AttributeError: 'ParsedRequirement' object has no attribute 'req'
```

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,19 +1,17 @@
...
 _version_re = re.compile(r'__version__\s+=\s+(.*)')
 
 with open('shipment_management/__init__.py', 'rb') as f:
-    version = str(ast.literal_eval(_version_re.search(
-        f.read().decode('utf-8')).group(1)))
+	version = str(ast.literal_eval(_version_re.search(f.read().decode('utf-8')).group(1)))
 
-requirements = parse_requirements("requirements.txt", session="")
+with open('requirements.txt') as f:
+	install_requires = f.read().strip().split('\n')
```
Comment (DeveloperMetal) [inline setup.py:14]: did they really remove requirement.txt parsing?
maybe it got moved to another api?

**Comment by doniafrizal** [comment]
i tried to install, but still error error due to pip.req, hope the setup.py fix can use asap.

**Comment by Alchez** [comment]
@doniafrizal, are you facing that using the v11 branch on this app?

**Comment by doniafrizal** [comment]
> @doniafrizal, are you facing that using the v11 branch on this app?

I am using v13

**Comment by Alchez** [comment]
@doniafrizal oh no, I meant the branch on this app, are you using `master`, `hotfix`, `v11`, etc?

**Comment by doniafrizal** [comment]
Oh sorry. I Have forked Your hotfix branch and installed oN my ERPNext v13 after modified The setup.py.

**Pull request merged**